### ISSUE TITLE
Someone using the HX-DOS version? not working here

### ISSUE THREAD

**Issue by TheElf01** [joncampbell123/dosbox-x#3961]
### Question

Hi, greetings. I tested many versions of dosbox-x  always same problem

<a href="https://ibb.co/B3XrfQ3"><img src="https://i.ibb.co/xLpf2BL/dos.jpg" alt="dos" border="0"></a>

I have a celeron 366mhz, 256mb ram,  DOS 7.1 FAT32 HDD, Riva 128 4mb, SB16 ISA

Using regular dosbox on hx-dos extender works OK

Any dieas are welcome! thanks

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by Torinde** [comment]
@KormaX-of-DOS

**Comment by maron2000** [comment]
HX-DOS version is confirmed to run on 32bit VS version before it is uploaded.

**Comment by maron2000** [comment]
Confirmed that it launches from on DOS booted on VMware using Win98 SE boot disk.
![hx-dos](https://github.com/joncampbell123/dosbox-x/assets/68574602/dbbad729-b568-4494-9bab-73666225d51f)

**Comment by TheElf01** [comment]
> Confirmed that it launches on DOS booted on VMware using Win98 SE boot disk. ![hx-dos](https://user-images.githubusercontent.com/68574602/265914301-dbbad729-b568-4494-9bab-73666225d51f.png)

thanks for testing,  where is the lastest package to download? i want to test but is not in github or webpage  thanks!

**Comment by maron2000** [comment]
I don't see any special configuration in config.sys.
Unpack the zip file to your hard drive, boot from Win98 SE boot disk, and run DOSBox-X.
You may want to launch DOSBox-X with default configuration `dosbox-x -defaultconf`

![configsys](https://github.com/joncampbell123/dosbox-x/assets/68574602/6039903f-30f9-42b6-86ca-25e5002d922c)

**Comment by TheElf01** [comment]
> I don't see any special configuration in config.sys. Unpack the zip file to your hard drive, boot from Win98 SE boot disk, and run DOSBox-X. You may want to launch DOSBox-X with default configuration `dosbox-x -defaultconf`
> 
> ![configsys](https://user-images.githubusercontent.com/68574602/267871557-6039903f-30f9-42b6-86ca-25e5002d922c.png)

Hi thanks,  i test with a empty drive, just  DOS7 system, minimal config.sys with exactly your config (himem+common),  fresh dosbox-x  and black screen, nothing

I tried with -defaultconf but same, just black screen


Then, i tried something i did not before...   remove the sound card,  and worked. 


I remove soundcard in the 3 computers, and work fine in all,    in the Pentium 3 that have a PnP card, i just did not initialize (i use UNISOUND) and dosbox work fine,  but in others, that have no PnP sound card, even without set blaster, if sound card is pressent present dosbox from start

I tried changing IRQ in sound card,  5,7,  nothing, same black screen


Any ideas?  thanks for your time

**Comment by maron2000** [comment]
VMware has SB16 emulation.
![sb](https://github.com/joncampbell123/dosbox-x/assets/68574602/db8e6be7-f3c0-4bd1-9ba3-8e1b73d1e337)

**Comment by TheElf01** [comment]
> VMware has SB16 emulation. ![sb](https://user-images.githubusercontent.com/68574602/267925964-db8e6be7-f3c0-4bd1-9ba3-8e1b73d1e337.png)

No, i say in your real computer, in VM work, is in real hardware not working

thanks

**Comment by TheElf01** [comment]
> VMware has SB16 emulation. ![sb](https://user-images.githubusercontent.com/68574602/267925964-db8e6be7-f3c0-4bd1-9ba3-8e1b73d1e337.png)

Sorry i delete my latest post,    i did not explain well,  i say what sound card you use in real hardware to test, thanks!

I did not have any luck to make work,  only without sound

**Comment by rderooy** [comment]
Do a search for "HX sound" in the closed issues... Seems it can be made to work with virtualbox at least. I have heard say that the VMware SB16 emulation is rather poor.

**Comment by maron2000** [comment]
Sure, I'm not recommending running on VMware, but it is enough to confirm that HX-DOS version runs on DOS.
Not only sound but sometimes mouse goes crazy.
